bleachbit/Common.py

```diff
@@ -107,6 +107,10 @@
 # system cleaners
 if sys.platform.startswith('linux'):
     system_cleaners_dir = '/usr/share/bleachbit/cleaners'
+
+if sys.platform.startswith('darwin'):
```

Comment:
To avoid repeating the same directory, you could also use an `or` condition after `startswith(`linux`)`